runtime/runtime.go

```
@@ -195,3 +214,14 @@ func truthy(val Value) bool {
 		return val != nil
 	}
 }
+
+func evalBlock(block *ast.BlockExpr, local map[string]Value) Value {
```

golangci-lint (1.64.8)

func `evalBlock` is unused

(unused)